testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -92,4 +102,17 @@ class ProfileTestHelper @Inject constructor(
     testCoroutineDispatchers.runCurrent()
     return result
   }
+
+  /** ObserveForever is needed in order to complete the Action which returns the LiveData.
```

Comment:
This should describe what the helper does, then why it's needed. The 'observeForever' is irrelevant--that's an implementation detail. All callers care about is when & why they should call this, so we should answer that.

Reply:
Done, let me know if it is correct. 

```kdocs
  /**
   * While performing any action based on the LiveData, this helper function should be used.
   * Without this helper function the LiveData cannot be accessed.
   *
   * @param data is the LiveData which needs to accessed while performing action.
   */
```

Reply:
Left a comment regarding this directly. Thanks!